src/AccessibilityInsights/MainWindow.xaml

```diff
@@ -458,7 +458,7 @@
                                                 <MenuItem.Header>
                                                     <TextBlock Name="tbTimer">
                                                     <Run Text="{x:Static properties:Resources.tbTimerText1}"/>
-                                                    <TextBox TextChanged="tbxTimer_TextChanged" Name="tbxTimer" PreviewTextInput="tbxTimer_PreviewTextInput" Text="5" MaxLength="2" FontSize="11" Width="16" Margin="2,-4" 
+                                                    <TextBox TextChanged="tbxTimer_TextChanged" Name="tbxTimer" PreviewTextInput="tbxTimer_PreviewTextInput" Text="{x:Static properties:Resources.MainWindow_NumberFive}" MaxLength="2" FontSize="11" Width="16" Margin="2,-4" 
```

Comment:
The value here happens to be 5, but it could be anything from 2 to 99. With that in mind, it might make sense to define this in terms of intent instead of value. Maybe `MainWindow_DefaultTimerDelayInSeconds` or similar?